pkg/kubectl/cmd/convert/convert.go

```diff
@@ -76,8 +76,8 @@ type ConvertOptions struct {
 	genericclioptions.IOStreams
 }
 
-func NewConvertOptions(ioStreams genericclioptions.IOStreams) *ConvertOptions {
-	return &ConvertOptions{
+func newConvertOptions(ioStreams genericclioptions.IOStreams) *Options {
```

Comment:
What is the reason for changing this from public to private?

Reply:
Agreed, it doesn't seem to be used anywhere else in the project. I guess it comes down to what our strategy is for making kubectl functionality usable by other applications, and do we want to expose this as the interface for creating options.

I will defer to others on the project about whether we want the NewXxxxxxxxxOptions() to be public or not.  Just the the other comment, it seems like they are public everywhere else.